frontend-react/src/shared/ContentSection/ContentSection.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from "classnames";
+
+import { Tile } from "../Tile/Tile";
+
+export interface ContentSectionProps
+    extends React.PropsWithChildren<
+        React.HTMLAttributes<HTMLElement> & ContentItem
+    > {}
+
+export const ContentSection = ({
+    title,
+    summary,
+    items,
+    className,
+    children,
+    ...props
+}: ContentSectionProps) => {
+    const classnames = classNames("usa-section", className);
```

Comment:
I don't think we should be passing down any css classes, and I don't think they're even being used here? If we have styles / functionality that are toggled on or off, they should be passed down as props.

Reply:
Will fix